code/modules/mob/living/living_say.dm

```diff
@@ -105,7 +105,7 @@ GLOBAL_LIST_EMPTY(channel_to_radio_key)
 	return returns
 
 
-/mob/living/say(message, verb = "says", sanitize = TRUE, ignore_speech_problems = FALSE, ignore_atmospherics = FALSE, ignore_languages = FALSE)
+/mob/living/say(message, verb = "says", sanitize = TRUE, ignore_speech_problems = FALSE, ignore_atmospherics = FALSE, ignore_languages = FALSE, is_ai_auto = FALSE)
```

Comment:
I'd prefer if this was `forced` like on TG, which will allow for a lot more flexibility in logging

Reply:
`forced` isn't suitable, since the whole point here is to distinguish (server) automated from (admin) forced. I changed it to `automatic`, which is at least a bit more generic.